SimPEG/electromagnetics/static/resistivity/simulation_2d.py

```diff
@@ -648,25 +734,61 @@ def bc_type(self, value):
             "bc_type", value, ["Neumann", ("Robin", "Mixed")]
         )
 
-    def getA(self, ky):
+    @property
+    def MeSigma2D(self):
+        """Inner product matrix on edges for 2D"""
+        if getattr(self, "_MeSigma2D", None) is None:
+            if len(self.sigma) == self.mesh.nC:
+                self._MeSigma2D = self.MeSigma
+            elif len(self.sigma) == 3 * mesh.nC:
+                self._MeSigma2D = self.mesh.self.mesh.get_edge_inner_product(
+                    model=self.sigma[:2*self.mesh.nC]
+                )
+            else:
+                raise NotImplementedError(
+                    "Only isotropic and linear isotropic conductivities implemented."
```

Comment:
This formulation should actually be able to handle all anisotropic conductivities (as we don’t need to invert the matrix). Let me double check what the tensor reduces to.

Reply:
Oh I just meant in the future it can support more than just axial anisotropy, compared to the cell centered one that would only support axial.